client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -623,6 +632,7 @@ export const useCommandsArrayWpcom = ( {
 					navigate( `/activity-log/${ site.slug }` );
 				},
 				filter: ( site: SiteExcerptData ) => ! isP2Site( site ) && ! isNotAtomicJetpack( site ),
+				filterNotice: __( 'Only listing sites with hosting features enabled.' ),
```

Comment:
The filter is hiding the P2s and Jetpack sites.
Simple sites has also the Activity log.

```suggestion
				filterNotice: __( 'Only listing sites hosted on WordPress.com.' ),
```

Reply:
Fixed with 2044d72984b